samples/copyclean.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+#    This file is part of python-registry.
+#    It tries to make a clean copy of hive file
+#    clean meaning without unused (not understood) data
+#
```

Comment:
windows leaves a lot of junk in there, and there is no documentation on not understood data.
Maybe something like this?
`It tries to make a clean copy of hive file removing parts that is not used (windows leaves some junk) This makes it easier to analyze real data in hives.`